config.template.yaml

```diff
@@ -0,0 +1,68 @@
+log:
+    pretty: true
+    level: debug
+dispatcher:
+    enabled: false
+    # required if dispatcher enabled
+    privatekey:
```

Comment:
any docs on generating this would be helpful, got any feedback?

also, the configured `privatekeys` are just a conditional in the bootstrap file, the private keys get generated randomly right now with `role.GenerateNewPeer()`. is this just temporary or should we also just use `os.Getenv("DISPATCHER_ENABLED")`, etc?

https://github.com/data-preservation-programs/ValidationBot/blob/main/cmd/bootstrap.go#L153